src/sql/workbench/services/connection/browser/connectionController.ts

```diff
@@ -47,24 +47,24 @@ export class ConnectionController implements IConnectionComponentController {
 			onCreateNewServerGroup: () => this.onCreateNewServerGroup(),
 			onAdvancedProperties: () => this.handleOnAdvancedProperties(),
 			onSetAzureTimeOut: () => this.handleonSetAzureTimeOut(),
-			onFetchDatabases: (serverName: string, authenticationType: string, userName?: string, password?: string, azureAccount?: string) => this.onFetchDatabases(
-				serverName, authenticationType, userName, password, azureAccount).then(result => {
+			onFetchDatabases: (serverName: string, authenticationType: string, userName?: string, password?: string, token?: string) => this.onFetchDatabases(
+				serverName, authenticationType, userName, password, token).then(result => {
 					return result;
 				}),
 			onAzureTenantSelection: (azureTenantId?: string) => this.onAzureTenantSelection(azureTenantId),
 		}, providerName);
 		this._providerName = providerName;
 	}
 
-	protected async onFetchDatabases(serverName: string, authenticationType: string, userName?: string, password?: string, azureAccount?: string): Promise<string[]> {
+	protected async onFetchDatabases(serverName: string, authenticationType: string, userName?: string, password?: string, token?: string): Promise<string[]> {
 		let tempProfile = this._model;
 		tempProfile.serverName = serverName;
 		tempProfile.authenticationType = authenticationType;
 		tempProfile.userName = userName;
 		tempProfile.password = password;
 		tempProfile.groupFullName = '';
 		tempProfile.saveProfile = false;
-		tempProfile.azureAccount = azureAccount;
+		tempProfile.azureAccount = token;
```

Comment:
Repurposing this to hold any token. Renaming `azureAccount` seems like a somewhat sizeable change.